player/playloop.c

```diff
@@ -612,7 +612,11 @@ static void handle_heartbeat_cmd(struct MPContext *mpctx)
         double now = mp_time_sec();
         if (mpctx->next_heartbeat <= now) {
             mpctx->next_heartbeat = now + opts->heartbeat_interval;
-            system(opts->heartbeat_cmd);
+            int res = system(opts->heartbeat_cmd);
+            if (res != 0) {
+                MP_WARN(mpctx, "Heartbeat command exited with status %d!\n",
+                        WEXITSTATUS(res));
```

Comment:
You could add Windows ifdeffery. I'd just drop it. Actually, I'd drop the whole feature (it's 100% redundant with Lua scripting), but who knows who would complain.